homebrew_challenge/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./Header.css";
+import {
+  PAGE_ABOUT_US,
+  PAGE_ACCESSIBILITY,
+  PAGE_DATA_SOURCES,
+  PAGE_OVERVIEW,
+  PAGE_REGIONAL,
+} from "../../pages/PageConsts";
+
+function Header({ pageTitle }, { setCurrentPage }) {
```

Comment:
I'm surprised this works - does clicking the STA icon still return to the overview page?
I think it should be:
`function` Header({ pageTitle, setCurrentPage }) {`